viewing_party/main.py

```diff
@@ -0,0 +1,129 @@
+# Wave 1 s
+def create_movie(movie_title,genre,rating):
+    movie = {}
+    if movie_title == None or genre == None or rating == None:
+        return None
+    else: 
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating
```

Comment:
this works just fine! one thing to think about. What happens if the variables aren't valid? now you've got `movie` store in memory just to have it be trashed. so maybe think about this:

```suggestion
    if movie_title == None or genre == None or rating == None:
        return None
    else: 
        movie = {"title" movie_title, "genre": genre, "rating": rating}
```